src/views/new-evaluation/newEvaluation.js

```diff
@@ -0,0 +1,302 @@
+const NAV = function (option = {}){
+    return `
+    <ul class="header headerT">
+        ${logoButton(option.Logo)}
+        ${constructNav(option)}
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
```

Comment:
Didn't I see this stuff once before? Remember, always try to keep your code DRY! ;)